test/directives/faDeckSpec.js

```diff
@@ -0,0 +1,279 @@
+          );
+        $scope.views = [0, 1, 2];
+        $scope.openVal = true;
+        $scope.$apply();
+
+        var deck = $famous.find('fa-deck')[0].renderNode;
+
+        expect(deck.isOpen()).toEqual(true);
+
+        common.destroyApp(app);
+      });
+    });
+
+    describe("close", function(){
+      it("using a direct value - fa-open=\"false\"", function() {
+        var app = common.createApp(
+            '<fa-deck fa-pipe-from="eventHandler" fa-open="false">' +
+              '<fa-view ng-repeat="view in views">' +
+                '<fa-surface fa-pipe-to="eventHandler" fa-size="[undefined, 100]"></fa-surface>' +
+              '</fa-view>' +
+            '</fa-deck>'
+          );
+        $scope.views = [0, 1, 2];
+        $scope.$apply();
+
+        var deck = $famous.find('fa-deck')[0].renderNode;
+
+        expect(deck.isOpen()).toEqual(false);
+
+        common.destroyApp(app);
+      });
+
+      it("using the return value of a function - fa-open=\"open()\"", function() {
+        var app = common.createApp(
+            '<fa-deck fa-pipe-from="eventHandler" fa-open="open()">' +
+              '<fa-view ng-repeat="view in views">' +
+                '<fa-surface fa-pipe-to="eventHandler" fa-size="[undefined, 100]"></fa-surface>' +
+              '</fa-view>' +
+            '</fa-deck>'
+          );
+        $scope.views = [0, 1, 2];
+        $scope.open = function(){
+          return false;
+        }
+        $scope.$apply();
+
+        var deck = $famous.find('fa-deck')[0].renderNode;
+
+        expect(deck.isOpen()).toEqual(false);
+
+        common.destroyApp(app);
+      });
+
+      it("using a function name - fa-open=\"openFunc\"", function() {
+        var app = common.createApp(
+            '<fa-deck fa-pipe-from="eventHandler" fa-open="openFunc">' +
+              '<fa-view ng-repeat="view in views">' +
+                '<fa-surface fa-pipe-to="eventHandler" fa-size="[undefined, 100]"></fa-surface>' +
+              '</fa-view>' +
+            '</fa-deck>'
+          );
+        $scope.views = [0, 1, 2];
+        $scope.openFunc = function(){
+          return false;
+        }
+        $scope.$apply();
+
+        var deck = $famous.find('fa-deck')[0].renderNode;
+
+        expect(deck.isOpen()).toEqual(false);
+
+        common.destroyApp(app);
+      });
+
+      it("using a $scope variable - fa-open=\"openVal\"", function() {
+        var app = common.createApp(
+            '<fa-deck fa-pipe-from="eventHandler" fa-open="openVal">' +
+              '<fa-view ng-repeat="view in views">' +
+                '<fa-surface fa-pipe-to="eventHandler" fa-size="[undefined, 100]"></fa-surface>' +
+              '</fa-view>' +
+            '</fa-deck>'
+          );
+        $scope.views = [0, 1, 2];
+        $scope.openVal = false;
+        $scope.$apply();
+
+        var deck = $famous.find('fa-deck')[0].renderNode;
+
+        expect(deck.isOpen()).toEqual(false);
+
+        common.destroyApp(app);
```

Comment:
There is `common.destroyApp(app)` at the end of most of the tests: maybe we can consider using `afterEach()`?